docs/_sources/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
this should stay in `.github/ISSUE_TEMPLATE/bug_report.md`

Reply:
should be fixed :+1: